packages/guideline-blocks-settings/src/components/Attachments/types.ts

```diff
@@ -16,7 +16,7 @@ export type AttachmentsProps = {
     onDelete: (attachmentToDelete: Asset) => void;
     onUpload: (uploadedAttachments: Asset[]) => Promise<void>;
     onBrowse: (browserAttachments: Asset[]) => void;
-    onSorted: (sortedAttachments: Asset[]) => void;
+    onSorted?: (sortedAttachments: Asset[]) => void;
```

Comment:
Why this change? As it still uses the `useAttachments` Hook, its still provided always.

Reply:
Initially I wanted to remove this, but realised that would be a breaking change, and since it will be reintroduced soon I did not want to go that way.
I marked is as optional because at the moment this function does not do anything.

Reply:
Then I wouldn't mark it optionally as it would be a breaking changes if we add it later on again